modules/lib/lib-event/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+    "compilerOptions": {
+        "lib": ["es6"],
```

Comment:
why is this es6?

Reply:
Doesn't Nashorn support at least `find` and `Map` features? Just a plain old ES5 with `const`? If not, then it must be `es5`, I guess.

Reply:
> Doesn't Nashorn support at least find and Map features?

No :(

Reply:
Okay, will change it to "es5" then.